pkg/didcomm/protocol/issuecredential/service.go

```diff
@@ -346,12 +346,7 @@ func nextState(msg service.DIDCommMsg, outbound bool) (state, error) {
 
 // canTriggerActionEvents checks if the incoming message can trigger an action event
 func canTriggerActionEvents(msg service.DIDCommMsg) bool {
-	switch msg.Type() {
-	case ProposeCredentialMsgType, OfferCredentialMsgType, RequestCredentialMsgType, IssueCredentialMsgType:
-		return true
-	}
-
-	return false
+	return msg.Type() == ProposeCredentialMsgType || msg.Type() == OfferCredentialMsgType
```

Comment:
I understand the choreography diagram in the RFC has the Issuer automatically sending the credential when they receive `request-credential`, but really `propose-credential` and `offer-credential` are optional messages (see [here](https://github.com/hyperledger/aries-rfcs/blob/master/features/0036-issue-credential/README.md#messages)). This means `request-credential` is a valid entrypoint.

Reply:
Done! I have added `RequestCredentialMsgType`